src/components/sidebar/Sidebar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './Sidebar.css'
+
+function Sidebar() {
+  return (
+    <>
+      <nav className="sidebar">
+        <div className="nav-links">
+          <a className="sidebar-link">Link1</a>
+          <a className="sidebar-link">Link2</a>
+          <a className="sidebar-link">Link3</a>
+          <a className="sidebar-link">Link4</a>
+
+        </div>
+
+
+      </nav>
+    </>
+  )
+}
+
+export default Sidebar
```

Comment:
Use only named exports. It will be easier when application grows big and debugging.